pkg/upgrade/upgrade.go

```diff
@@ -232,6 +233,38 @@ func UpdateFromLegacyVersion(cli client.Client, platform deploy.Platform, appNS
 	return nil
 }
 
+// CleanupDeprecatedFeatures deletes deprecated features by removing the corresponding FeatureTrackers
+func CleanupDeprecatedFeatures(ctx context.Context, cli client.Client, platform deploy.Platform, appNamespace string) error {
+	crd := &apiextv1.CustomResourceDefinition{}
+	if err := cli.Get(ctx, client.ObjectKey{Name: "featuretrackers.features.opendatahub.io"}, crd); err != nil {
+		return err
+	}
+
+	toDelete := []*featuresv1.FeatureTracker{
+		featuresv1.NewFeatureTracker("kserve-temporary-fixes", appNamespace),
+	}
+
+	var multiErr *multierror.Error
+	for _, featureTracker := range toDelete {
```

Comment:
maybe i did not get the whole picture here, but why not just call  removeFeatureTracker() or even Cleanup()?

Reply:
Cleanup sounds like a good way of doing it. Thanks for the feedback, I was having trouble trying to figure out the `Feature`/`FeatureTracker` APIs